go.mod

```diff
@@ -193,7 +193,7 @@ replace (
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.7
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
-	github.com/evmos/ethermint => github.com/yihuang/ethermint v0.6.1-0.20230118154933-386780367ed1
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.2-cronos
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/386780367ed1...v0.20.2-cronos

Reply:
might miss [validate](https://github.com/evmos/ethermint/blob/6b8297b9dbf8139b58ef8ac9ae48f5c539e2a077/server/start.go#L224) before startTelemetry

Reply:
I think it's alright, we only use `cfg.Telemetry.Enabled` field here, don't care about the other fields.